### ISSUE TITLE
Package brew not installed(Mac when I pay the VPN)

### ISSUE THREAD

**Issue by Ajrocwuu** [sentinel-official/sentinel#52]

**Comment by riotcs** [comment]
Can you explain the problem more clearly? Did you follow the setup guide throughout?

**Comment by Arsca** [comment]
Hi,

Chiming in here as I have the same issue.

I have the same problem on my MacOS: 10.13.5 (17F77)
1) Activate the Testnet switch
2) Click to connect to one of the VPN points on the list
3) In the dialog, press connect to chosen VPN end point
4) Balck "Package Brew is not Installed" alert panel appear.

**Comment by fabulous345** [comment]
Did you install homebrew according to the setup guide?

**Comment by tonyxia2016** [comment]
Got the same issue here. Sentinel 0.0.5 alpha, MacOS 10.14.  
Always show "Package brew not installed" when I click a VPN. Homebrew installed, and upgraded to the latest version. What else do I need to do? Where is the install document for Mac?

**Comment by tonyxia2016** [comment]
Follow the install help https://github.com/sentinel-official/sentinel/tree/master/wallet and install homebrew again https://www.howtogeek.com/211541/homebrew-for-os-x-easily-installs-desktop-apps-and-terminal-utilities/ , the issue is still there.    Sentinel 0.0.5 alpha, MacOS 10.14.

**Comment by ymai6** [comment]
一样的问题

**Comment by thenikin** [comment]
this is still happening to me. Latest mac os, homebrew installed. Any hints?

**Comment by octalmage** [comment]
So I was having the same issue, and it was because I did not open the package like the instructions state and run the binary directly! After doing that, the homebrew error went away but then I got an error related to not being able to install OpenVPN. 

After this I decided to close everything, open the terminal and run Sentinel directly using sudo. This was the full command:

```
sudo /Applications/sentinel.app/Contents/MacOS/sentinel
```

After doing this everything is working as expected! 

![Screen Shot 2019-06-02 at 2 36 10 PM](https://user-images.githubusercontent.com/142006/58766408-c707f000-8543-11e9-93e8-fa1cae728df0.png)

I hope this helps! Great work so far on the app, looks fantastic!
